arm-dependencies.sls

```diff
@@ -0,0 +1,89 @@
+arm-dependencies:
+  pkg.installed:
+    - pkgs:
+      - g++-aarch64-linux-gnu
+      - g++-arm-linux-gnueabihf
+
+{% set path = 'https://servo-rust.s3.amazonaws.com/ARM/' %}
+{% set targets = [{ 'name': 'arm-linux-gnueabihf', 
+                    'unknown_name': 'arm-unknown-linux-gnueabihf',
+                    'version': 'v1',
+                    'file': 'armhf-trusty-libs.tgz',
```

Comment:
Looks like the `file` variables are redundant here with `local_name`. I think `local_name` is clearer, so let's keep those ones.